src/inc/til/bitmap.h

```diff
@@ -83,14 +83,8 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
 
             void _calculateArea()
             {
-                // Backup the position as the next one.
-                _nextPos = _pos;
-
-                // Seek forward until we find an on bit.
-                while (_nextPos < _end && !_values[_nextPos])
-                {
-                    ++_nextPos;
-                }
+                const auto nextPos = _pos == 0 ? _values.find_first() : _values.find_next(_pos - 1);

```

Comment:
I would prefer a comment left above this explaining what it's doing in plain language. It requires a bit of mental effort to decode the ternary and remember/lookup how `find_first` and `find_next` operate. With algorithms such as these, I like to leave too much explanation of why/how they work so a reader at a later date can understand what to care about and what not to care about without having to cross-reference anything.